### DISCUSSION TITLE
Migrate images to helmfile/helmfile namespace

### DISCUSSION THREAD

**Discussion by naphta** [helmfile/helmfile#202] [Ideas]
I can see there's been some movement around using github actions and releasing into github packages but I presume there is a desire to move to distinguish this fork from roboll/helmfile at this point?

Happy to go ahead and help out but would love to know if there's a strategy or plan in place already before potentially putting together a PR which steps on the toes of some other work. I did search around a little but couldn't see much to this tune.

**Comment by yxxhero** [comment]
@naphta what is your plan?
